src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -26,18 +26,17 @@ When you migrate your users to this model, benefits include:
 
 ## Requirements [#requirements]
 
-Before using [the user migration wizard](#find), please review the guidelines below. The wizard is aimed at migrating users for fairly simple New Relic organizations. We'll continue to add to the user migration features so that it works better for more complex organizations. 
+Before using [the user migration wizard](#find), please review the guidelines below. 
 
 Requirements and guidelines: 
 
 * You must have the [Owner role](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/#roles) to start using the user migration wizard. 
-* If your organization has more than 20 accounts, we recommend **not** using the wizard at this point in time. Note that this refers to having 20 or more accounts, not users.
-* If your organization has a fairly complex [parent/child-account structure](/docs/accounts/original-accounts-billing/original-users-roles/mastersub-account-structure), we recommend **not** using the wizard at this point in time. 
-* If you frequently [add accounts](/docs/accounts/accounts-billing/account-structure/mastersub-account-structure/#creating) to your organization, we recommend **not** using the wizard at this point in time. Currently, with the new model, you must get New Relic assistance to add accounts. 
-* Roles related to user management (ability to add and update users, change user type, create access grants) are currently applied for an entire organization and can't be assigned at the account level. If you require account-level user management role assignment, we recommend not using the wizard. 
-* If you were previously using OneLogin, Azure, or Okta apps for a New Relic integration, note that you'll likely need a different, newer app titled "New Relic by organization" (and not "New Relic by account").
+* If your organization has more than 20 accounts, we recommend **not** using the user migration wizard. **You can disregard this requirement if you're being helped by New Relic representatives.** (Note that this refers to having 20 or more accounts in your [organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#new-model), not users.)
```

Comment:
Is there a next step for orgs that have 20+ accounts, but no rep?

Reply:
No, no next steps really. It kind of preselects for companies that have reps.